main.nf

```diff
@@ -833,8 +833,9 @@ workflow cellranger_count_libraries {
 
 workflow cellranger_count_demuxlet {
     include {
-        demuxlet as DEMUXLET;
-    } from './workflows/popscle' params(params)
+        cellranger_dataChannel_to_bam_barcodes;
```

Comment:
I would use only one naming style. Having `dataChannel` is this really needed ? 
What about `cellranger_output_to_bam_barcodes` ?

Reply:
Thanks, I made these updates in the last commit.